Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook

```diff
@@ -554,7 +554,7 @@
+                                "query": "let selectedCategories = dynamic([{selCategory}]);\r\nlet selectedTotSev = dynamic([{selTotSev}]);\r\nSqlAssessment_CL\r\n| where _ResourceId =~ {VmResourceId:value}\r\n| extend asmt = parse_csv(RawData)\r\n| extend AsmtId=tostring(asmt[1]), CheckId=tostring(asmt[2]), DisplayString=asmt[3], Description=tostring(asmt[4]), HelpLink=asmt[5], TargetType=case(asmt[6] == 1, \"Server\", asmt[6] == 2, \"Database\", \"\"), TargetName=tostring(asmt[7]), \r\nSeverity=case(asmt[8] == 30, \"High\", asmt[8] == 20, \"Medium\", asmt[8] == 10, \"Low\", asmt[8] == 0, \"Information\", asmt[8]==1, \"Warning\", asmt[8]==2, \"Critical\", \"Passed\"), Message=tostring(asmt[9]), TagsArr=split(tostring(asmt[10]), \",\"), Sev = toint(asmt[8])\r\n| where AsmtId == \"{AsmtId}\" \r\nand (set_has_element(dynamic([{filterCheckId}]), CheckId) or \"{filterCheckId}\" == \"'*'\")\r\nand (set_has_element(dynamic([{filterTargetName}]), TargetName)  or \"{filterTargetName}\" == \"'*'\")\r\nand set_has_element(dynamic([{filterSev}]), Sev)\r\nand (array_length(set_intersect(TagsArr, dynamic([{filterTags}]))) > 0 or \"{filterTags}\" == \"'*'\")\r\n| extend Category = case(array_length(set_intersect(TagsArr, dynamic([{typeSystem:value}]))) > 0, '0',\r\narray_length(set_intersect(TagsArr, dynamic([{typeAdmins:value}]))) > 0, '1',\r\narray_length(set_intersect(TagsArr, dynamic([{typeDevs:value}]))) > 0, '2',\r\n'3')\r\n| where (Sev >= 0 and array_length(selectedTotSev) == 0 or Sev in (selectedTotSev))\r\nand (Category in (selectedCategories) or array_length(selectedCategories) == 0)\r\n| project TargetType, TargetName, Severity, Tags=strcat_array(array_slice(TagsArr, 1, -1), ', '), CheckId, SeverityCode = toint(Sev), Message\r\n| distinct * \r\n| summarize Cnt = count() by SeverityCode, Severity, Tags, CheckId\r\n| project Severity, Tags, CheckId, Cnt, SeverityCode\r\n| order by SeverityCode desc, Cnt desc\r\n\r\n",
```

Comment:
Why are modifying SQL VM notebook?

Reply:
There was some spacing issue bug that was reported. ​ The list of tags was not getting separated by spaces.
The change is visible in the screenshot.